app/build.gradle

```diff
@@ -7,7 +7,12 @@ apply from: "$project.rootDir/tools/gradle/versionCode.gradle"
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
-apply plugin: 'com.huawei.agconnect'
+
+// Apply AGConnect plugin only for Hvr builds
+if (getGradle().getStartParameter().getTaskRequests()
+        .toString().contains("Hvr")){
```

Comment:
Nit: use a single line. Also I think we should better use `toLowerCase` and then compare to "hvr" just in case. seems more futureproof.

Reply:
BTW I haven't tested it yet, but I'll do before giving the final approval

Reply:
Thank you for the feedback! I think [pattern matching](https://docs.groovy-lang.org/latest/html/documentation/#_find_operator) would be more flexible:
`(getGradle().getStartParameter().getTaskRequests().toString() =~ /(h|H)vr/)` 
My worry with lowercase string search is it can accidentally match with other strings like `...hVr...`, for instance `...HighVr...` (made up example).
I am partial towards matching, but will make changes using `toLowerCase` if that's more in line with the requirement.